src/backend/aspen/cli/sync_auth0.py

```diff
@@ -17,7 +18,98 @@
     session_scope,
     SqlAlchemyInterface,
 )
-from aspen.database.models import Group, User
+from aspen.database.models import Group, Role, User, UserRole
+
+
+# This isn't following the same conventions as the rest of the syncer classes :(
+class RoleManager:
+    def __init__(self, db: Session, a0: Auth0Client, dry_run: bool):
+        self.db = db
+        self.a0 = a0
+        self.dry_run = dry_run
+        self.groupmap: Dict[str, Group] = {}
+        self.rolemap: Dict[str, Role] = {}
+
+    def load_context(self):
+        roles = self.db.execute(sa.select(Role)).scalars().all()
+        self.rolemap = {role.name: role for role in roles}
+
+        groups = self.db.execute(sa.select(Group)).scalars().all()
+        self.groupmap = {group.auth0_org_id: group for group in groups}
```

Comment:
why not just call load_context in __init__?

Reply:
Doing work in constructors (especially interacting with external systems, like a db in this case) can make testing more difficult, so I usually try to avoid it.